Turkey/Program.cs

```diff
@@ -167,14 +168,17 @@ public static async Task<string> GenerateNuGetConfigIfNeededAsync(string additio
                 }
 
                 string nugetConfig = null;
-                try
+                if (useSourceBuildNuGetConfig)
```

Comment:
Can you remove from this method all code that is no longer used (like this branch, and also the previous branch `if (netCoreAppVersion.Major < 4)`), and all methods which were only called by these branches?

Reply:
I think the first part of this relates to https://github.com/redhat-developer/dotnet-regular-tests/pull/252/files#r1228201263 and we will need to bring it back in some form.

Filed https://github.com/redhat-developer/dotnet-bunny/issues/76 for `netCoreAppVersion.Major < 4` and related code.

Reply:
> I think the first part of this relates to https://github.com/redhat-developer/dotnet-regular-tests/pull/252/files#r1228201263 and we will need to bring it back in some form.

I prefer to remove unused code. We can always add it back when needed.